app/src/main/java/com/github/sdp/mediato/utility/adapters/CollectionAdapter.java

```diff
@@ -35,24 +35,22 @@ public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
         .inflate(R.layout.layout_profile_movie_item, parent, false);
 
     return new ViewHolder(view);
-
   }
 
   @Override
   public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
     Review review = collection.getReviewsList().get(position);
     Media media = review.getMedia();
-    holder.mediaImage.setImageResource(R.drawable.bg_movie2);
+
+    // Set the media title
     holder.mediaTitle.setText(media.getTitle());
-    try {
-      int grade = review.getGrade();
-      holder.mediaRating.setText(getStarString(grade));
 
-    } catch (Exception e) {
-      throw new RuntimeException(e);
-    }
-    /* holder.mediaRating.setText(review.getGrade());*/
+    // Set the media rating
+    int grade = review.getGrade();

```

Comment:
Is it possible that this may cause some error ? Since the grade attribute isn't mandatory in the declaration of a new review getGrade() might return 0 when the user simply didn't give a rating. Maybe we can choose not to display any rating when it's less than 1.

Reply:
True, I changed it to display an empty string if the return value is 0.